### ISSUE TITLE
Can't connect from PC, but can from android app - same opvn file

### ISSUE THREAD

**Issue by PickledNewt** [pivpn/pivpn#389]
Hi team, 

I have setup OpenVpn on my Pi. I can connect from my android mobile using the OpenVpn app and certificate below just fine. However, when I try from my Windows 10 laptop, I just can't get it to work. 

I have installed OpenVpn, and pasted the client.ovpn file below (without the bits *'d) into the config folder. Nothing else. When I run it however, I keep getting a message saying "openvpn tls key negotiation failed to occur within 60 second".

I have turned off the firewall on my laptop and run the file as Administator from CMD prompt, but I keep getting the same result.

Is there something else I should be doing?

Thank you all for any help you can provide in advance. 
James.

###########################################

client
dev tun
proto udp
remote ****.com 3***
resolv-retry infinite
nobind
persist-key
persist-tun
key-direction 1
remote-cert-tls server
tls-version-min 1.2
verify-x509-name server name
cipher AES-256-CBC
auth SHA256
comp-lzo
verb 1
<ca>
-----BEGIN CERTIFICATE-----
****
-----END CERTIFICATE-----
</ca>
<cert>
-----BEGIN CERTIFICATE-----
****
-----END CERTIFICATE-----
</cert>
<key>
-----BEGIN ENCRYPTED PRIVATE KEY-----
****
-----END ENCRYPTED PRIVATE KEY-----
</key>
<tls-auth>
#
# 2048 bit OpenVPN static key
#
-----BEGIN OpenVPN Static key V1-----
****
-----END OpenVPN Static key V1-----
</tls-auth>

**Comment by cfcolaco** [comment]
Please read issue #386 and check all the things mentioned there... you can also do a search in the already closed // opened tickets and check those stuff... if that doesn't work please come back saying what have you tried and not! =)

**Comment by PickledNewt** [comment]
Thank you.... will give that a go :)

**Comment by cfcolaco** [comment]
@PickledNewt any news?

**Comment by PickledNewt** [comment]
@cfcolaco Yes and no....

I tried various different approaches from that thread, and looking around, but couldn't get it to work. I then decided to try my wife's certificate (which _I believe_ I set up in the exact same way at the start, and which also works fine on her phone...) and hey presto, it worked perfectly first time on my computer. Tried again with mine, and same error message. 

I don't know what is different between the two, but I am happy using hers on my computer....

Thank you for your help and swift response, it was much appreciated.
